test/jdk/java/awt/regtesthelpers/Util.java

```diff
@@ -663,4 +663,13 @@ public void run() {
         return result.get(0);
     }
 
+    /**
+     * Checks if the current system is running on Wayland display server on Linux.
+     *
+     * @return {@code true} if the system is running on Wayland display server
+     */
+    public static boolean isOnWayland() {
+        return System.getenv("WAYLAND_DISPLAY") != null;
+    }
+
```

Comment:
Is it better to have this function in `jdk/test/lib/Platform.java` since we have all the other OS related checks in Platform.java ?

Reply:
Yes, that sounds right to me. Updated.